### ISSUE TITLE
Sequence of DID based approache

### ISSUE THREAD

**Issue by sebastiankb** [w3c/wot-discovery#201]
I am trying to understand section 5.5, but it is not clear what the sequence is. It seems that a Thing has a DID. How is this announced and where do I get this DID document? I think it would be great to have a sequence diagram such as Figure 2 and 3.

**Comment by mmccool** [comment]
I don't think we need a sequence diagram since it would replicate what is in the DID spec, but I think we should briefly explain that a DID can be resolved into a DID document (using resolution mechanisms dependent on the particular DID method identified in the DID itself) and that DID document can have a service entry providing a URL that can be used as an introduction.  The trouble with a sequence diagram is that there are over a hundred DID methods currently proposed and they might use very different approaches to resolution.  We DO want to recommend some specific DID resolution mechanisms (for security) but that should be done in wot-security-best-practices since it can change over time.

Note that DID is an intro mechanism though and by design it is not security-sensitive.  Sensitive PII should not be placed in the DID or the DID document, though.

**Comment by mmccool** [comment]
I think there is some general cleanup needed in introductions and a summary of the DID resolution process could be added as part of that.

**Comment by mmccool** [comment]
Changed resolution target for this as "resolve by PR" since should only require changes to informative text.  Also DID was published, which make it easier to justify including it.  The "resolution" process is in fact DID-method-specific but this is a "feature" of the DID spec, so we will just have to deal with that...
